cmd/thanos/rule.go

```diff
@@ -73,10 +73,10 @@ func registerRule(m map[string]setupFunc, app *kingpin.Application) {
 		Default("1m"))
 	evalInterval := modelDuration(cmd.Flag("eval-interval", "The default evaluation interval to use.").
 		Default("30s"))
-	tsdbBlockDuration := modelDuration(cmd.Flag("tsdb.block-duration", "Block duration for TSDB block.").
```

Comment:
Just curious what happened with those? The above `eval` still have modelDuration

Reply:
Thanks for pointing out! `.Int64()` is definitely wrong as the is not an integer, I need to use `Duration()`. And to answer why not `modelDuration`, we need these as int64 at the end, so having it as `model.Duration` will lead to lot of type conversions.